pkg/katautils/config.go

```diff
@@ -356,6 +352,12 @@ func (h hypervisor) getInitrdAndImage() (initrd string, image string, err error)
 		return "", "", errors.New("either image or initrd must be defined in the configuration file")
 	}
 
+	if errInitrd != nil {
```

Comment:
This doesn't feel right to me, according to your commit message. This looks like if either fails then we return an error, and I think the caller will check that and fail?
Should we not return an error only if both image and initrd error out? I might be missing some subtlety though.

Reply:
@grahamwhaley : How about ...

```
if (image != "" && errImage != nil)  {
		return initrd, image, errImage
} else if ( initrd != "" && errInitrd != nil) {
             	return initrd, image, errInitrd
}
```

Reply:
@grahamwhaley : We should fail if

1. Both are defined.
2. Both are not defined.
3. One which is defined, that file does not exist.